src/main/java/edu/hawaii/its/api/util/JsonUtil.java

```diff
@@ -44,6 +49,25 @@ public static <T> List<T> asList(final String json, Class<T> type) {
         return result;
     }
 
+    public static String readJsonFileToString(String filePath) {
+        String result = null;
+        try {
+            Resource resource = new ClassPathResource(filePath);
+            result = Files.readString(resource.getFile().toPath());
+        } catch (Exception e) {
+            logger.error("Error: " + e);
+        }
+        return result;
+    }
```

Comment:
Let's make this a private function since it would only be used as a helper function in this class.

Also it would be cool if it threw an error if the file was not a json. We can simply just check the file ends in .json.

Reply:
okk!! That's great